tests/examples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+if(BUILD_EXAMPLES)
+    if(BUILD_TESTING)
+	set(EXAMPLE_BINARY_DIR ${CMAKE_BINARY_DIR}/examples)
+	add_test(
+	    NAME "ExampleTesting"
+	    COMMAND sh -c "MOCOCRW_EXAMPLE_BINARY_PATH=${EXAMPLE_BINARY_DIR} MOCOCRW_SRC_DIR=${EXAMPLE_DIR} /usr/bin/python3 -m unittest"
```

Comment:
I don't think this will work when cross compiling with `CMAKE_CROSSCOMPILING_EMULATOR` set unless you modify the python script to start the target binary with the emulator manually.

Reply:
What's your suggestion here?
Shall I check if CMAKE_CROSSCOMPILING_EMULATOR is enabled and abort the tests?
I guess that execute the examples using "qemu user mode" might be quite cumbersome as it is not installed on every system.

Reply:
Or, detect that, pass the value of CMAKE_CROSSCOMPILING_EMULATOR to the python script and support invoking things through that.

We already execute all other tests like that when cross-compiling, so if any tests should pass when cross-compiling, this setup is already required. If we could skip the Python script, CMake would do that for us, too.